### ISSUE TITLE
SuperOTC should work with Windows 10 "Creator's Update" (15063) too

### ISSUE THREAD

**Issue by Artoria2e5** [adobe-fonts/source-han-serif#18]
@imbushuo reported that he is using SuperOTC smoothly on 15063. The "which file to use" part of the READMEs for Source Han Sans and Source Han Serif should be updated on the next release.

**Comment by kenlunde** [comment]
Thank you. This is already on the proverbial radar, so I will close this issue.

**Comment by Artoria2e5** [comment]
Thanks for acknowledging this. Just tested SuperOTC on my copy of 15063, and verified that it works.

**Comment by kenlunde** [comment]
Although I don't use Windows, I am very, very glad that Windows will finally provide complete support for the Super OTCs of the _Source Han_ typeface families. Eventually my hope is that there will be a single deployment format: the Super OTC. It provides support for apps that don't support language tagging by having the fonts for each language available, and at the same time supports apps that do support language tagging (meaning that one needs to work with a single-language font, then language-tag for the other languages).

As a side note, if you inspect the collateral that I prepared, in terms of the fonts that are embedded in the PDFs, you'll notice that I used only the _SourceHanSans-*_ and _SourceHanSerif-*_ fonts, and language-tagged to get the Korean, Simplified Chinese, and Traditional Chinese glyphs to appear correctly.
